### ISSUE TITLE
SUV vs Coupe/Sedan Performance

### ISSUE THREAD

**Issue by FARQ3X** [fkk-cz/noire_vehicles#1657]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Certain vehicles are extremely overpowered when they should not be (E.G Sup G Class Mercedes costs 169k and is MUCH MUCH faster than M5 CS or Police Explorer vs Police Mustang, the Mustang is super slow compared to what it should be, while the explorer which should be heavy and slow and mostly used for traffic stops is used in chases as it is faster and better than the Charger and Mustang in most situations.) 

### Expected Behavior

I think a handling rebalance is in order as SUVs are dominant in almost ALL situations. I understand the developers have much bigger issues on their mind, but I would be more than happy to do testing in SP and slowly dripfeed fix the handling. I DO NOT THINK TOP SPEEDS ARE TOO LOW, matter of fact I feel the top speeds are very very good. The game is too small for higher speeds, however acceleration and turning etc do need tweaks.

### Steps To Reproduce

PDM -> Supporter Cars -> G Class 169k vs M5 CS(Called Competition even though F90 M5 is a competition), G Class vs almost anything basically
Same thing with non sup cars
Jeep Grand Cherokee SRT 392 vs Challenger Hellcat -> Grand Cherokee wins in straight line AND turns.

### Anything else?

Im more than happy to do testing, send clips and let you guys make the decision all I would do is help rebalance as I really enjoy your city!

**Comment by CCPPEN** [comment]
+1 I agree 👍

**Comment by Blase295** [comment]
Agreed

**Comment by LPLips7** [comment]
I Agree too, Good idea

**Comment by AkumaCx** [comment]
Should do 👍

**Comment by akTommy** [comment]
This needs to happen. Great Idea

**Comment by fkk-cz** [comment]
Ideally coupes and sedans should have better handling & performance compared to their SUV equivalents, including high performance ones such as the Trackhawk. Note that vehicle adjustments have been in progress for months, there are bound to be issues like this that will eventually get resolved. A little bit of patience & understanding will help here.

**Comment by FARQ3X** [comment]
Zonda Tricolore needs a nerf.... same with Maserati Gran Turismo!

**Comment by radoncombe** [comment]
> Zonda Tricolore needs a nerf.... same with Maserati Gran Turismo!

I disagree with you about the Zonda....

**Comment by FARQ3X** [comment]
> > Zonda Tricolore needs a nerf.... same with Maserati Gran Turismo!
> 
> I disagree with you about the Zonda....

The zonda is a 3.6s to 60 vehicle with a top speed of 217 MPH irl. The car is nowhere near as capable as for example a Bugatti Chiron, Hennessy Venom GT, Koenigsegg Agera RS, Veyron, Hell id bet a 765LT would smoke a Zonda. The Zonda R is fast but even then itrs slower than any of those cars. Its an 11 second 1/4 mile at best. All the other cars are sub 10 second.

**Comment by BalakayXO** [comment]
Multiple vehicles do not perform as expected. A Nissan skyline r32 STOCK hits 158 mph. Yet the speed isn’t consistent. I don’t expect it to be entirely perfect but a STOCK Honda civic is faster than a stock bmw m3.

**Comment by BalakayXO** [comment]
F90 M5 *Non Supporter* Needs a buff.
F90 M5 Comp *F90 Supporter* Needs a buff.
F90 M5 CS *M5 Competition in Supporter* Needs a big buff (These cars are much faster than Hellcats in all areas just based on gearing alone)
765LT Needs a buff.*Gets smoked by 720s, 458 Prior, 600LT and multiple others even though its the worlds meta supercar atm*
Agera RS needs a fat buff
Venom GT needs a fat buff
850CSi needs a nerf

**Comment by BalakayXO** [comment]
As an ASE certified mechanic I’d happily test cars

**Comment by LPLips7** [comment]
IMO, after I tested those BMW's, the Agera RS and the Venom GT that you listed as vehicles need a buff, they were hitting the speeds they should. For a 500k car, the agera RS performs heaps well, and when i tested the Venom GT, it went a decent speed. The BMW F90 M5 non supporter was performing fine aswell... And also the 765 LT is heaps fast already man. That shit zooming

**Comment by FARQ3X** [comment]
> 

I disagree personally. The Daytona Charger will outpace the BMWs which would never happen irl. The Agera RS should perform near enough a Chiron and it does not even though it is LIGHTER with similar power, Venom GT should perform similar to Agera. The 765LT is an embarrassment tbh. I get smoked by 720s all day. I truly think the Charger Daytona is the sweetspot balance wise for the city, handling is what it should be. The M5s have been known as supercar killers since launch, yet they cant smoke a Hellcat (W211 Mercedes Chassis from early 2000s with a big Hemi up front and a ton of weight), Agera RS was the worlds fastest car for a hot minute yet is outperformed by the likes of a Zonda Tricolore which is more a piece of art than performance machine? 765LT should be smoking a P1 from a roll and 100% be smoking a 720s AND Senna from launch yet is dead even with a 600LT and smoked by its baby brother the 720s? +1 On all F90s getting a buff +1 on 765LT, Agera RS and Venom, 850CSi I think just needs a top speed nerf tbh. Performance irl isnt based on price buddy. Paganis are expensive even though they underperform for example. Many cars are more art pieces.

**Comment by FARQ3X** [comment]
Bugatti, Pagani are less hypercar than art tbh. Keonigsegg is all about practical use of the car, not 1 thought about "Artistic"
765LT is the fastest production car in the world yet gets smoked by its baby brothers from McLaren.
Destroys a 911 Turbo S in a roll, It cannot hang at launch tbf.
https://www.youtube.com/watch?v=czLv8DYQ1s0
Here it destroys an F8, GT2 RS from launch too....
https://www.youtube.com/watch?v=mhyny8vjBDU
Only way a Chiron will destroy it is from a roll, just because of the weight,

Here is a good article explaining how weight, and lying about real performance which is typical of BMW and McLaren is not reflected irl.
https://www.carscoops.com/2021/05/could-the-mclaren-765lt-actually-be-quicker-than-the-p1/

**Comment by LPLips7** [comment]
If you gonna spend 1.6 million compared to a 500k Agera then of course its going to out-perform it. Also I've had a Chiron and the handling on that thing is fucking insane. The Agera RS is a measly 500k compared to 2.4 million dollars. Of course its not going to have the same performance. For example, the Chiron has a decent speed, but it sacrifices acceleration and a higher top speed for amazing grip, cornering and over-all handling. The Rimac C two is 2 mil. It has amazing acceleration, but tops out at 180 and has mid armour and handling. While yes, the performance of a vehicle isn't purely based on its Price, you have to remember a Daytona is 95k and a Demon is 300k / the new demon is 1 mil. I do think the 720 does need a nerf though. Also the Pagani Zonda has set many speed and acceleration development break - throughs.

**Comment by LPLips7** [comment]
Also a Bugatti Chiron would smack the shit out of a GT2 Performance wise. 0 - 60MPH in under 2.6 seconds, hitting up to 250 base model (this is all IRL). Remember, this is a game. Also how would Koenigsegg be a car of "Practical Use" when it was just stated that the Agera RS was basically once one of the fastest cars in the world, and now with the jesko and the jesko absolut. The jesko is one of the fastest cars in the world, and is a peice of art aswell.

**Comment by LPLips7** [comment]
This would be similar, if i was to say the 600k Veyron should be going up to 240 MPH flat, because its a Veyron, not considering its cheap price

**Comment by FARQ3X** [comment]
> If you gonna spend 1.6 million compared to a 500k Agera then of course its going to out-perform it. Also I've had a Chiron and the handling on that thing is fucking insane. The Agera RS is a measly 500k compared to 2.4 million dollars. Of course its not going to have the same performance. For example, the Chiron has a decent speed, but it sacrifices acceleration and a higher top speed for amazing grip, cornering and over-all handling. The Rimac C two is 2 mil. It has amazing acceleration, but tops out at 180 and has mid armour and handling. While yes, the performance of a vehicle isn't purely based on its Price, you have to remember a Daytona is 95k and a Demon is 300k / the new demon is 1 mil. I do think the 720 does need a nerf though. Also the Pagani Zonda has set many speed and acceleration development break - throughs.

The Pagani Zonda has set 0 Speed and Acceleration development breakthroughs. It set a new standard for carbon fiber that is all. Matter of fact that the Zonda is just a Mercedes V12 with a manual just makes you sound even less car inclined. Let me put it in layman terms: You can put 50k into a Civic and it will smoke a Daytona/Zonda in a straight line even though Daytona/Zonda will be more expensive. If you are spending 1.6 Mil on a car you arent paying for perfomance, you are paying for looks, refinement and prestige. A Senna/P1 will smoke a 765LT on the track(Debatable since McLaren is so scared to release 720s numbers to not embarass Senna), not in a straight line that is 1000%
Bugatti Chiron would smack the shit out of a GT2 in a straight line. It would not beat it on a track. Keonigseggs body parts are all functional, Interior is all function and weight saving. Jesko literally is all about Aerodynamics.
600k Veyron should have a similar top speed to Chiron with much slower acceleration and even shittier handling.
https://supercarlists.com/fastest-cars-quarter-mile
See how the Chiron is only .4 Seconds faster to a 1/4 mile than a 765LT and only .5 seconds faster than Veyron? Chiron is art, It is all luxury when you get inside, a 765LT is a Racing Seat with a monocog strapped to it.

**Comment by LPLips7** [comment]
Im still not quite sure you understand, its a game, and if i spend 2.4 million on a Chiron compared to a 600k Veyron then no shit its going to perform better in game. If we are talking "IRL" since for some reason you dont understand its a game, The Agera wouldn't be 500k, and if it was, it wouldn't perform like it does IRL. (Same concept in game..). Also the Zonda had a fucking quick acceleration on it man. Zonda is both, a piece of art IMO, and has some speed to it.

**Comment by FARQ3X** [comment]
Zonda is slow, has been ever since the 2000s. Its all looks no go. Again complain all you want, but IMO price should not have ANYTHING to do with perfomance. By your logic price dictates performance because cars are a dick swinging competition of who can spend the most in your eyes. If someone spends 5m on a Chiron irl, they dont care that a venom GT is catching up, because a Venom GT is just a Chevy V8, with 2 turbos strapped to a Lotus. Its singular purpose is speed. Bugattis purpose is being the most exclusive vehicle possible.

**Comment by LPLips7** [comment]
I think you're making very valid points here, and tbh i hope actually upgrading or modifying vehicles aka being able to choose race and other sort of cool varying modifications, can come to the mechanics update. This would then, make it alot more fun. But, Im assuming an admin may or may not explain it to you in more depth, or from a different POV. Also the Jesko is a v8 aswell. (V8 Twin- Turbocharged). It also may depend on if the vehicle is super charged, or turbo charged, as they are 2 different things.

**Comment by LPLips7** [comment]
Also, some vehicles are still in the works of getting QOL adjusted, or handling adjusted

**Comment by FARQ3X** [comment]
> I think you're making very valid points here, and tbh i hope actually upgrading or modifying vehicles aka being able to choose race and other sort of cool varying modifications, can come to the mechanics update. This would then, make it alot more fun. But, Im assuming an admin may or may not explain it to you in more depth, or from a different POV. Also the Jesko is a v8 aswell. (V8 Twin- Turbocharged). It also may depend on if the vehicle is super charged, or turbo charged, as they are 2 different things.

Supercharger vs Turbo is very very simple. Supercharger causes parasitic loss(Runs off timing belt) Turbos use waste gas for extra power. Upgrading/Modifying vehicles isnt the problem, certain cars are simply much slower than other cars just because they have been overnerfed/underperform with current handling files, and again the F90 M5s all need a boost IMO, These cars aren't perfect, they weigh alot and will never corner anywhere as well as a McLaren/Ferrari etc. But in a 1/4 mile they will hang in there for much much longer than a Daytona. a Daytona in perfect conditions would probably do 11.2 maybe 11.1 sec 1/4 mile, due to RWD and crappy old Chassis it will not be consistent. An F90 M5 will do it in 10.8-10.9 consistently yet in the city the Hellcats will race past the M5 with very little problem. I do not own any Koenigsegg or BMW, I own a Charger because I truly think its handling is the most balanced in city and I like drifting that bitch :D. I own the 765LT and getting embarassed by the 720s pisses me off but I wasnt really gonna mention it. I know cars are getting QOL adjusted... hence why I replied and gave reasons for every vehicle, I do not expect admins to spend all day testing every car! Simply of the opinion that we should as a community help out instead of arguing because we prefer this car/this car. Base it on IRL so its nice and fair :)

**Comment by LPLips7** [comment]
I think if the 720s is 60k less than the 765lt and is going faster, it needs a nerf. I also think that sometimes things can happen to vehicles after QOL nerfs. for e.g. When the aston martin Valkyrie was nerfed, it has now capped at 150 no upgrades. It should be making at least 180 - 185 with no upgrades and 190 -200 with upgrades. (Its a 3 mil dollar Vehicle). If i ever feel a vehicle has been nerfed Too much in a certain area of that car, (Considering the price and what vehicle it is) I would make a ticket

**Comment by FARQ3X** [comment]
> I think if the 720s is 60k less than the 765lt and is going faster, it needs a nerf. I also think that sometimes things can happen to vehicles after QOL nerfs. for e.g. When the aston martin Valkyrie was nerfed, it has now capped at 150 no upgrades. It should be making at least 180 - 185 with no upgrades and 190 -200 with upgrades. (Its a 3 mil dollar Vehicle). If i ever feel a vehicle has been nerfed Too much in a certain area of that car, (Considering the price and what vehicle it is) I would make a ticket
720s is fine the way it is IMO. Its the 765LT that needs a slight buff in launch. The Valkyrie is not a top speed vehicle, its a track car, very very different approach to the gearing. Definitely needs a buff from 150 to 160-170 (Ive noticed 2/3 of real life performance is roughly what translates into the city) but its realistically just an F1 car made street legal, so its acceleration below 120MPH is what its all about and corners, not highway cruising per se like Bugatti/765LT. McLarens track car is the Senna, Bugattis is the Pur Sport, highway cruiser/top speed is the SS 300. We are using this github instead of tickets to try get more feedback from community! The admins have final say either way!

**Comment by LPLips7** [comment]
I do believe vehicle should handle better such as those mclarens, and the valkyrie. Yes- The bugatti pur sport is the track vehicle, while it has a speed limiter limiting it to below 250mph irl, it has amazing handling, and is the sport version. Track Mclarens that are street legal IRL sacrifice comfort, but have heeps good cornering and speed, compared to a chiron IRL which has a higher top speed, 10/10 comfort, but isn't as good on handling and turning compared to the mclarens.

**Comment by Competent1** [comment]
Going to be closing this as this has turned into an argument/heated discussion and others saying "buff this" and "buff that" when these vehicles were already tweaked for QOL Adjustments. That's not the point of the initial issue. Regarding the initial issue, I believe most vehicles mentioned in here have been tweaked. If there are vehicles with issues, do make another GitHub issue regarding it. Thanks!
